api/tests/data.py

```diff
@@ -5,6 +5,7 @@
 from api.models.program import Conference, Presentation
 from api.models.program import Place, Category, Difficulty
 from api.models.profile import Profile
+from api.models.sponsor import Sponsor, SponsorLevel
```

Comment:
SponsorLevel을 사용하지 않고 있네요

```suggestion
from api.models.sponsor import Sponsor
```